batch/batch/front_end/front_end.py

```diff
@@ -1285,7 +1310,7 @@ async def ui_get_billing(request, userdata):
 async def _query_billing_projects(db, user=None, billing_project=None):
     args = []
 
-    where_conditions = []
+    where_conditions = ["billing_projects.status != 'deleted'"]
```

Comment:
Eventually, will you have a test that ensures the rest endpoint get_billing_project fails if the project is deleted?

Reply:
Good call. I'll add that in the other PR.